crates/katana/executor/src/implementation/sir/mod.rs

```diff
@@ -159,13 +161,13 @@
                     crate::utils::log_events(receipt.events());
 
                     if let Some(reason) = receipt.revert_reason() {
-                        info!(target: "executor", "transaction reverted: {reason}");
+                        info!(target: LOG_TARGET, reason = %reason, "Transaction reverted.");
                     }
 
                     ExecutionResult::new_success(receipt, trace, fee)
                 }
                 Err(e) => {
-                    info!(target: "executor", "transaction execution failed: {e}");
+                    error!(target: LOG_TARGET, error = %e, "Executing transaction.");
```

Comment:
similar to [comment](https://github.com/dojoengine/dojo/pull/1702#discussion_r1547225893)